src/ChannelServer/Network/ChannelPacketSender.cs

```diff
@@ -1510,7 +1510,239 @@ public static void ZC_PC_MOVE_STOP(Character character, Position position, Direc
+			packet.PutInt(id);
+			packet.PutFloat(1);
+			packet.PutBinFromHex("01 39 EC C0");
+			packet.PutFloat(1);
+			packet.PutFloat(position1.X);
+			packet.PutFloat(position1.Y);
+			packet.PutFloat(position1.Z);
+			packet.PutFloat(position2.X);
+			packet.PutFloat(position2.Y);
+			packet.PutFloat(position2.Z);
+
+
+			character.Map.Broadcast(packet, character);
+		}
+
+		/// <summary>
+		/// Creates a skill in client
+		/// </summary>
+		/// <param name="character"></param>
+		/// <param name="id"></param>
+		/// <param name="position"></param>
+		/// <param name="direction"></param>
+		/// <param name="create"></param>
+		public static void ZC_NORMAL_Skill(Character character, int id, Position position, Direction direction, bool create)
+		{
+			var actorId = 1234; // ActorId (entity unique id for this skill)
+			var skillState = 0; // skillState seems to be an ENUM of animation states (0 = initial animation, 1 = touched animation)
+			var distance = 20.0f; // Distance to caster? Not sure. Observed values (20, 40, 80)
+
+			var packet = new Packet(Op.ZC_NORMAL);
+			packet.PutInt(0x57);
+			packet.PutInt(character.Handle);
+			packet.PutBinFromHex("11 18 27 00"); // Heal skill effect 
+			packet.PutInt(id); // SkillId
+			packet.PutInt(1); // Skill Level ?
+			packet.PutFloat(position.X);
+			packet.PutFloat(position.Y);
+			packet.PutFloat(position.Z);
+			packet.PutFloat(0.707f); // Direction
+			packet.PutFloat(0.707f); // Direction
+			packet.PutInt(0);
+			packet.PutFloat(distance);
+			packet.PutInt(actorId); 
+			packet.PutByte(create);
+			if (!create)
+			{
+				skillState = 1;
+			}
+			packet.PutInt(skillState);
+			packet.PutInt(0);
+			packet.PutInt(0);
+			packet.PutInt(0);
+			packet.PutInt(0);
+			packet.PutInt(0);
+			packet.PutInt(0);
+			packet.PutInt(0);
+
+			character.Map.Broadcast(packet, character);
+		}
+
+		/// <summary>
+		/// Creates a skill in client
+		/// </summary>
+		/// <param name="character"></param>
+		/// <param name="actorId"></param>
+		/// <param name="enable"></param>
+		public static void ZC_NORMAL_ParticleEffect(Character character, int actorId, int enable)
+		{
+			var packet = new Packet(Op.ZC_NORMAL);
+			packet.PutInt(0x61);
+			packet.PutInt(actorId);
+			packet.PutInt(enable);
+
+			character.Map.Broadcast(packet);
+		}
+
+		/// <summary>
+		/// Unkown purposes when casting skills
+		/// </summary>
+		/// <param name="character"></param>
+		/// <param name="id"></param>
+		/// <param name="position"></param>
+		/// <param name="direction"></param>
+		public static void ZC_NORMAL_Unkown_1c(Character character, int id, Position position, Direction direction)
+		{
+			var packet = new Packet(Op.ZC_NORMAL);
+			packet.PutInt(0x1c);
+			packet.PutByte(0);
+			packet.PutBinFromHex("9F D2 42 0B");
+			packet.PutInt(id); // Used to be skillId, but observed 00 00 00 00 now
+			packet.PutFloat(position.X);
+			packet.PutFloat(position.Y);
+			packet.PutFloat(position.Z);
+			packet.PutFloat(0.707f);
```

Comment:
Direction is in the parameters, because the packet will need it. Line 1617 and 1618. I used constants (0.707f) for testing purposes. We don't know the purpose of that packet yet, and don't know what direction to send from the function calling it.